Anchorage.swift

```diff
@@ -33,139 +33,145 @@ extension NSLayoutDimension : LayoutAnchorType {}
 extension NSLayoutXAxisAnchor : LayoutAnchorType {}
 extension NSLayoutYAxisAnchor : LayoutAnchorType {}
 
-// MARK: - Equality Constraints
+#if swift(>=3.0)
+
+    // MARK: - Equality Constraints
```

Comment:
Oh, maybe it's intentional, because of the `if Swift` check? If so, feel free to disregard